README.md

```diff
@@ -28,6 +38,29 @@ await pushNotificationReceived(snapyrData);
 // after receiving a push interaction callback, track tapped metric to Snapyr
 const snapyrData = notification.data?.snapyr;
 await pushNotificationTapped(snapyrData);
+
+// --- In-App messaging section (optional) ---
+
+// listen to in-app messages from Snapyr 
+onSnapyrInAppMessage((inappMessage: SnapyrInAppMessage) => {
+    const actionToken = inappMessage.actionToken;
+    // example... stash this message in a useState so we can read its properties elsewhere
+    setCurrentInAppMessage(inappMessage);
+    if (inappMessage.actionType == "custom" && inappMessage.content.payloadType == "html") {
```

Comment:
At one point we had talked about the SDK automatically filtering out "non-custom" for the actions that get handed to the app. Did we decide against that? No biggie either way, just might be confusing to see an example here of filtering on custom if they will never see any other actionType.

Reply:
That's right, good catch! Only custom actions trigger the callback. I'll update the example